comms/xlog/Portfile

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                xlog
+version             2.0.14beta4
+
+categories          comms
+license             GPL-3
+platforms           darwin
+maintainers         nomaintainer
```

Comment:
I do not feel I am currently able to take the responsibility. In the event that I get myself more organized in the future, I will take maintainership of the ports that I have contributed before then.